lib/SILOptimizer/Transforms/DeadCodeElimination.cpp

```diff
@@ -450,7 +415,8 @@ void DCE::propagateLiveness(SILInstruction *I) {
 SILBasicBlock *DCE::nearestUsefulPostDominator(SILBasicBlock *Block) {
   // Find the nearest post-dominator that has useful instructions.
   auto *PostDomNode = PDT->getNode(Block)->getIDom();
-  while (PostDomNode && !LiveBlocks.count(PostDomNode->getBlock()))
+  while (PostDomNode && (!PostDomNode->getBlock() ||
```

Comment:
Why was this changed? I am not saying it is incorrect. It just needs a comment at least explaining what the change is.